src/PostgREST/Config.hs

```diff
@@ -132,15 +140,21 @@ readOptions = do
       return appConf
 
  where
+  parseJwtAudience :: Name -> C.ConfigParserM StringOrURI
+  parseJwtAudience k =
+    preview stringOrUri . fromMaybe ("" :: Text) <$> C.key k >>= \case
```

Comment:
If the key is not present, you are assuming `""` (empty string).  This means that the JWT will be required to
have an audience claim being the empty string.  Perhaps not what is intended?

If you want postgrest to *require* the administrator to set the  audience claim, you should probably require the config be specified, rather than defaulting it to empty string.

Otherwise you should propagate the evidence that no audience predicate was specified (e.g. as a
`Nothing :: Maybe StringOrURI`) and if it was not specified, set the audience predicate to `(const True)` to
accept any audience.

Reply:
Done. Thanks for the feedback.